.travis.yml

```diff
@@ -22,8 +22,8 @@ sudo: false
 #dist:
```

Comment:
got it. The Makefile.am apparently still refers to cmake files (that are not there), so it can't "make dist" and "distcheck". Clean out those references and it should be good. Normally configure should have handled this IIRC...